app/utils/encrypt.js

```diff
@@ -0,0 +1,17 @@
+const { DEFAULT_SALT_ROUNDS } = require('../constants');
+const bcrypt = require('bcryptjs'),
+  {
+    common: {
+      api: { saltRounds = DEFAULT_SALT_ROUNDS }
```

Comment:
It's okay but is a little tricky to define a default value, i prefer something like 
``` 
const config = require('../../config');
const saltRounds = config.salt || DEFAULT_VALUE;
```